seleniumJSHttpESModuleEnvTest/src/test/scala/org/scalajs/jsenv/selenium/LocationTest.scala

```diff
@@ -0,0 +1,14 @@
+package org.scalajs.jsenv.selenium
+
+import org.junit.Assert._
+import org.junit.Test
+
+import scala.scalajs.js.Dynamic.global
+
+class LocationTest {
```

Comment:
I don't think we need this test twice. Depending on whether you do the one/two project thing, remove, or it's already there :)

Reply:
Sure, will not be a problem once I use a single project.